bazel/repositories.bzl

```diff
@@ -349,6 +350,16 @@ def _com_github_cyan4973_xxhash():
         actual = "@com_github_cyan4973_xxhash//:xxhash",
     )
 
+def _com_github_rshriram_sqlparser():
+    _repository_impl(
+        name = "com_github_rshriram_sqlparser",
+        build_file = "@envoy//bazel/external:sqlparser.BUILD",
```

Comment:
put the BUILD into https://github.com/envoyproxy/sql-parser?

Reply:
That project uses make files and has bunch of dependencies. I dont want to spend the effort to bazel-fy that project when that code there is going to move into envoy repo in few months [after cleanups, etc.].

If you give me a buildfile to stick into that repo with all the right bazel incantations, I would happily add it. If its just moving this build file into that repo, it kind of serves no purpose because this build file is a very simple stub.